substrate/primitives/staking/src/lib.rs

```diff
@@ -523,42 +486,6 @@ pub struct PagedExposureMetadata<Balance: HasCompact + codec::MaxEncodedLen> {
 	pub page_count: Page,
 }
 
-impl<Balance> PagedExposureMetadata<Balance>
```

Comment:
I am surprised we don't need these things? @Ank4n you would know better, they are related to paged exposures.